scripts/scripts.js

```diff
@@ -121,6 +121,78 @@ function decorateVideos(el) {
   });
 }
 
+function handlePost(response, message) {
+  if (!response.ok) {
+    message.innerText = 'Invitation Failed. Try Again.';
+  } else {
+    message.innerText = 'Invitation Successful!';
+  }
+}
+
+function submitInvite(event) {
+  // validate fields
+  let valid = 0;
+  const messageDiv = event.target.parentNode.parentNode.querySelector('.row.message');
+  const name = event.target.parentNode.parentNode.querySelector('.row > input[id=name]').value;
+  const email = event.target.parentNode.parentNode.querySelector('.row > input[id=email]').value;
+  const validRegex = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
+
+  if (name === '') {
+    messageDiv.innerText = 'Name is required';
+    valid += 1;
+  }
+  if (!email.match(validRegex)) {
+    messageDiv.innerText = 'Valid email is required';
```

Comment:
could we style the message div to show errors in red color, like in their current site?

Reply:
That's fair. I think the success message is in green too.